rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -24,9 +24,9 @@ docker_name="$FRAMEWORK_NAME-$CONTAINER_ID"
 
 mkdir -p "/tmp/pai-root/alive/$APP_ID"
 while /bin/true; do
-  touch "/tmp/pai-root/alive/$APP_ID/$CONTAINER_ID"
+  touch "/tmp/pai-root/alive/$APP_ID/yarn_$CONTAINER_ID"
   sleep 20
-  [ ! "$(docker ps | grep $docker_name)" ] && break
+  [ -f "/tmp/pai-root/alive/$APP_ID/docker_$CONTAINER_ID" ] && [ ! "$(docker ps | grep $docker_name)" ] && break
```

Comment:
can you remove /tmp/pai-root/alive/$APP_ID when exiting?

Reply:
`/tmp` will be cleaned by os regularly, yarn containers' runtime files are also stored there.

Reply:
ok. when cleaning, I assume os won't clean the most recent files (e.g., for those still running jobs)?

---
In reply to: [162090526](https://github.com/Microsoft/pai/pull/235#discussion_r162090526) [](ancestors = 162090526)